sa/sa.go

```diff
@@ -1480,6 +1540,39 @@ func (ssa *SQLStorageAuthority) GetOrderAuthorizations(
 	return byName, nil
 }
 
+// GetOrderForNames tries to find an order for the requested account ID that has
```

Comment:
"... tries to find an order with the exact set of names requested, associated with the given account ID." You can also remove "from the same account ID" from the next sentence.

Reply:
Clarified in a643602 - Thanks!